var/spack/repos/builtin/packages/root/package.py

```diff
@@ -428,8 +430,11 @@ def cmake_args(self):
 
         # Options related to ROOT's ability to download and build its own
         # dependencies. Per Spack convention, this should generally be avoided.
+
         options += [
-            define_from_variant("builtin_afterimage", "x"),
+            define_from_variant("builtin_afterimage", "x")
+            if "platform=darwin" not in self.spec
+            else define("builtin_afterimage", True),
```

Comment:
I think separately computing an `afterimage_enabled = "+x" in self.spec or "platform=darwin" in self.spec` boolean then doing a single `define("builtin_afterimage", afterimage_enabled)` option would be clearer.

Reply:
ok